composer.json

```diff
@@ -27,7 +27,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.2",
```

Comment:
Not need bump PHP version.

Reply:
Sorry, but why we don't need to update PHP version, if SensitiveParameter was added only in v. 8.2?

Reply:
I would be safely ignored in other versions.